packages/drivers/routerlicious-driver/src/wholeSummaryDocumentStorageService.ts

```diff
@@ -260,8 +260,17 @@ export class WholeSummaryDocumentStorageService implements IDocumentStorageServi
     private async initBlobCache(blobs: Map<string, ArrayBuffer>): Promise<void> {
         const blobCachePutPs: Promise<void>[] = [];
         blobs.forEach((value, id) => {
-            blobCachePutPs.push(this.blobCache.put(id, value));
+            const cacheKey = this.getBlobCacheKey(id);
+            blobCachePutPs.push(this.blobCache.put(cacheKey, value));
         });
         await Promise.all(blobCachePutPs);
     }
+
+    private getBlobCacheKey(blobId: string): string {
```

Comment:
getBlobCacheKey and getSnapshotCacheKey could be just one generic call to prepend docId.